bundle/internal/tf/schema/root.go

```diff
@@ -19,13 +19,17 @@ type Root struct {
 	Resource *Resources   `json:"resource,omitempty"`
 }
 
+const ProviderHost = "registry.terraform.io"
```

Comment:
Why break this out?

Reply:
In the terraform configuration itself we don't need to specify it, but in while preparing filesystem mirror with providers the host needs to be included in the folder structure. Having it here separately makes things a bit easier.